src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -2498,4 +2498,27 @@ public static void closeErrorDialog(RemoteRobot remoteRobot) {
             okButton.click();
         }
     }
+
+    /**
+     * Closes Terminal.
+     */
+    public static void closeTerminalTabs(RemoteRobot remoteRobot) {
+        ProjectFrameFixture projectFrame = remoteRobot.find(ProjectFrameFixture.class, Duration.ofSeconds(10));
+        try {
+            ProjectFrameFixture.rightClickOnTerminalTab(projectFrame);
+            ProjectFrameFixture.clickMenuOption(projectFrame, "action.CloseAllNotifications.text");
+
+            while (true) {
+                ComponentFixture terminateButton = projectFrame.find(ComponentFixture.class, byXpath("//div[@accessiblename='Terminate']"));
+                if (terminateButton.callJs("component.isEnabled();", false)) {
+                    terminateButton.click();
+                    // Optionally add a small delay here if needed
```

Comment:
I would like to see a small delay added here. The next test case is going to start almost immediately so make sure the terminate is complete.

Reply:
ok. Added delay.